### DISCUSSION TITLE
Has anyone been getting any interviews for Summer 2024 internships?

### DISCUSSION THREAD

**Discussion by kanjurer** [jenndryden/Canadian-Tech-Internships-and-New-Grad-2025#131] [General]
I have not received a single interview yet :'( even though I have been applying aggressively since June this year. Looking bleak 😿 . I guess I will do research next summer lol.

**Comment by ShevinuM** [comment]
> I have not received a single interview yet :'( even though I have been applying aggressively since June this year. Looking bleak 😿 . I guess I will do research next summer lol.

Glad to find that others are on the same boat. I think Canada has barely started with applications so maybe we are fine

**Reply by kanjurer** [reply]
Oh lol!!!!! I thought I was the only one!

**Reply by kanjurer** [reply]
Are people receiving OAs? I only got 1-2!

**Reply by KrishPatel13** [reply]
OAs ?

**Reply by kanjurer** [reply]
> OAs ?

Online Assessments

**Reply by ShevinuM** [reply]
> Are people receiving OAs? I only got 1-2!

I did get some OAs. Most US. Nothing from Stripe, Microsoft, DoorDash or Dropbox

**Comment by ChrisDryden** [comment]
The vast majority of Canadian interviews for summer internships do not happen until Nov-Jan. My understanding is that the interviews for the next summer that occur during this time-frame now are the most competitive, lowest volume and primarily US based. Due to the Co-Op System being so prevalent in Canada, I would argue that the majority of internship applications for the summer are actually opened in January for the beginning of the Spring semester.

**Reply by kanjurer** [reply]
Ah ok that explains it. Do you know when the big companies usually send out interviews in Canada, for Canada positions?

**Reply by ChrisDryden** [reply]
My understanding is that the three largest internship hiring by volume in Canada are RBC, Shopify and Amazon. RBC does its summer hiring around Mid November for Non-Coop and Late Jan-Early Feb for Co-op. Amazon has the majority of assessments and interviews from Oct-Nov but is continuous until wait-lists coming around January. Shopify does its recruiting around the Co-op time-frame and most of their interviews and assessments are around Late Jan-Early Feb.

I'm sure with how early you all are practicing Leet-code and beginning applications it will work out for you all.

**Reply by kanjurer** [reply]
TYSM ☀️, super helpful.

**Reply by jenndryden** [reply]
@ChrisDryden that is useful information, but I don't think Shopify has hired interns in a while

**Comment by tohmze** [comment]
I've gotten like 5 OAs but that's about it

**Reply by kanjurer** [reply]
That's a good amount

**Reply by tohmze** [reply]
yeah I bombed like 3 of them tho 😭

**Reply by kanjurer** [reply]
Bro 😭, at least you getting OAs

**Comment by ShevinuM** [comment]
How's the internship search going every one? It's dry season for me lol.

**Reply by kanjurer** [reply]
Super dry **crying noises**

**Reply by kanjurer** [reply]
If anybody wants to do leetcode together, I am down lol

**Reply by ShevinuM** [reply]
I'm down but I have a boatload of assignments. Maybe I'll hit you up some time in the future.

**Reply by kanjurer** [reply]
Bet, I am on my co-op so chilling lol

**Comment by kanjurer** [comment]
Has anyone heard from any of the big tech companies yet lol

**Reply by ShevinuM** [reply]
Yes, they thanked me for submitting the application. I'm so happy right now

(It's a joke)
